tests/datasets/claim_categorizer.yaml

```diff
@@ -307,8 +307,8 @@ dataset:
           expected_output:
             claim: "The associated legal structures were developed, for the most part, before the development of such information technologies as the postal service, telegraph, and telephone systems"
             claim_category: "established_reported_knowledge"
-            needs_external_verification: true
-            rationale: "This is a broad, background chronology claim: core U.S. criminal justice legal structures (rooted in English common law and the late-18th-century constitutional/statutory framework) predate the telegraph (1830s–1840s) and telephone (1870s)."
+            needs_external_verification: false
```

Comment:
Could we keep a complete rationale as before, but at the same time keep as not need external verification because it's common knowledge?

That way, we could keep important information as we had before, and at the same time raise the bar and avoid false positives.